halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/providers/openstack/OpenstackAccountValidator.java

```diff
@@ -1,54 +1,91 @@
 package com.netflix.spinnaker.halyard.config.validate.v1.providers.openstack;
 
+import com.netflix.spinnaker.clouddriver.consul.config.ConsulConfig;
 import com.netflix.spinnaker.clouddriver.openstack.config.OpenstackConfigurationProperties;
 import com.netflix.spinnaker.clouddriver.openstack.security.OpenstackNamedAccountCredentials;
 import com.netflix.spinnaker.halyard.config.model.v1.node.Validator;
 import com.netflix.spinnaker.halyard.config.model.v1.providers.openstack.OpenstackAccount;
 import com.netflix.spinnaker.halyard.config.problem.v1.ConfigProblemSetBuilder;
 import com.netflix.spinnaker.halyard.config.validate.v1.util.ValidatingFileReader;
 import com.netflix.spinnaker.halyard.core.problem.v1.Problem;
+import com.netflix.spinnaker.halyard.core.tasks.v1.DaemonTaskHandler;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
 import org.apache.commons.lang3.StringUtils;
-import org.springframework.stereotype.Component;
 
 import java.util.Arrays;
 import java.util.List;
 import java.util.stream.Collectors;
 
-@Component
-public class OpenstackAcountValidator extends Validator<OpenstackAccount> {
+@Data
+@EqualsAndHashCode(callSuper = false)
+public class OpenstackAccountValidator extends Validator<OpenstackAccount> {
+
+  final private List<OpenstackNamedAccountCredentials> credentialsList;
+
+  final private String halyardVersion;
+
   @Override
   public void validate(ConfigProblemSetBuilder psBuilder, OpenstackAccount account) {
-    String authUrl = account.getAuthUrl();
+    DaemonTaskHandler.message("Validating " + account.getNodeName() + " with " + OpenstackAccountValidator.class.getSimpleName());
+
+    String environment = account.getEnvironment();
+    String accountType = account.getAccountType();
     String username = account.getUsername();
     String password = account.getPassword();
     String projectName = account.getPassword();
     String domainName = account.getDomainName();
+    String authUrl = account.getAuthUrl();
+    List<String> regions = account.getRegions();
     Boolean insecure = account.getInsecure();
-    String userDataFile = account.getUserDataFile();
+    String heatTemplateLocation = account.getHeatTemplateLocation();
     OpenstackAccount.OpenstackLbaasOptions lbaas = account.getLbaas();
-    String regions = account.getRegions();
+    // TODO: consul config does not yet exist for OpenStack
```

Comment:
Where does it not exist? Could you link to an open bug in the component where it needs to be added in the `TODO()` braces?